credentials/apps/edx_credentials_extensions/edly_credentials_app/api/permissions.py

```diff
@@ -23,3 +24,16 @@ def has_permission(self, request, view):
             return True
 
         return False
+
+
+class CanAccessSiteCreation(permissions.BasePermission):
+    """
+    Checks if a user has the access to create and update methods for sites.
+    """
+
+    def has_permission(self, request, view):
+        """
+        Checks for user's permission for current site.
+        """
+        logger.info('User {user} is trying to access site creation API'.format(user=request.user.username))
+        return request.user.is_staff and request.user.username == EDLY_PANEL_WORKER_USER
```

Comment:
Kindly sync with @asadmanzoor93  on permission check. He's added the `or` check https://github.com/edly-io/edly-panel-edx-app/pull/127/files#diff-ec815efa63f0e0941b0096c4e4c42df09ebae7cbcbf9d2d018de41c90eb4dab9R55

Reply:
I've some concerns on that it is mandatory that our worker user will be always staff & username checks only that specific user can access this API. Adding `or` will allow any other staff user to access API.

Reply:
Updated.